DCA632FFFE049397.json

```diff
@@ -0,0 +1,17 @@
+{
+	"gateway_conf": {
+	 	"gateway_ID": "DCA632FFFE049397",
+		"servers": [ 
+			{ "server_address": "ttn-router-us-west",
```

Comment:
That's not a valid server address, check the list of servers here: https://www.thethingsnetwork.org/docs/gateways/packet-forwarder/semtech-udp.html#router-addresses